CONTRIBUTING.md

```diff
@@ -1,91 +1,101 @@
 # Contributing to Bisq
 
-Anyone is welcome to contribute to Bisq. This document provides an overview of how we work. If you're looking for somewhere to start contributing, see the [good first issue](https://github.com/bisq-network/bisq/issues?q=is%3Aopen+is%3Aissue+label%3A"good+first+issue") list.
+Read on to find out if taking part in the development of Bisq is for you and how to do it!
 
+## Are you "the guy"?
 
-## Communication Channels
+Java God, Bitcoin Native, Marketing Genius, Cypherpunk, Testing Machine, or simply someone who likes the idea of Bisq? Anyone is welcome to contribute! However, Bisq is no classic organization, there is no boss, there is no working hours and there is no classic payment. In short, Bisq is not for everyone.
 
-Most communication about Bisq happens in the [Bisq Slack workspace](https://bisq.network/slack-invite).
+**What is there to work on?** You have to be able to find and work a problem on your own. In general, we do not assign stuff to people, there is no specification to be implemented, there is no given way forward. You yourself have to find, choose and pick something to work on. If you get stuck, we are keen to help, but please do not wait for us to take your hand. Also, you can spend as much (or little) time as you want and when you want to - Bisq still is an open source project.
 
-Discussion about code changes happens in GitHub issues and pull requests.
+**Will you get paid?** Well, yes and no. With Bisq, there is no classic funding or investors, so there is no payout of fiat money.
+- Yes, because there is the DAO. The DAO allows for compensation requests to be filed and voted on. If such a voting goes in your favor, you receive BSQ, a colored bitcoin token. Sell your BSQ and you get your fiat eventually.
```

Comment:
The link is provided in the compensation section down below.

Besides, this basically says that there is the "thing" and it can pay you. Is there need for more? If you choose to read on, you will hit the details eventually.